src/strategies/galaxy-nft-with-score/index.ts

```diff
@@ -0,0 +1,98 @@
+import fetch from 'cross-fetch'
+
+export const author = 'albert'
+export const version = '0.1.0'
+
+const GALAXY_SUBGRAPH_URL = 'https://graphigo.prd.galaxy.eco/query'
+
+interface Config {
+  name: string,
+  weight: number,
+  cumulative: boolean
+}
+
+interface OwnerWithNfts {
+  owner: string,
+  nfts: {
+    id: string,
+    name: string
+  }[]
+}
+
+interface OwnerToNftCount {
+  [owner: string]: {
+    [name: string]: number
+  }
+}
+
+interface OwnerToScore {
+  [owner: string]: number
+}
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+) {
+  let fetchRes = await fetch(GALAXY_SUBGRAPH_URL, {
```

Comment:
Hello @alberthaotan It is better to use `subgraphRequest` here instead of `fetch`

Example 
https://github.com/snapshot-labs/snapshot-strategies/blob/2528c1bda2b025853bde6194e41004fe77bcafed/src/strategies/digitalax-mona-quickswap/index.ts#L31

Reply:
Sorry, this 'GALAXY_SUBGRAPH_URL' const var name is misleading, it's actually a graphql url, I changed the name to 'GALAXY_GRAPHQL_URL'. Thank you!